### ISSUE TITLE
Simplify `servicelogger.py`'s search for the servicevessel

### ISSUE THREAD

**Issue by aaaaalbert** [SeattleTestbed/nodemanager#119]
The servicelogger currently uses a rather complicated way to find out where it should put its logfiles to:
* The servicevessel public key is [hardcoded](https://github.com/SeattleTestbed/nodemanager/blob/master/servicelogger.py#L109-L110) (which is [a problem in itself](SeattleTestbed/custominstallerbuilder#13) when people fork their own Seattle-like testbed)
* It is looked up in the Seattle install's [`vesseldict`](https://github.com/SeattleTestbed/nodemanager/blob/master/servicelogger.py#L134)
* or, if that didn't work out, `.` (the current directory) is used.

The first two items could be implemented in a more general way by just `persist.restore_object`ing the `nodeman.cfg` that plainly states the name of the servicevessel.

The theoretical downside I can see is that if the servicevessel owner decided to give up the vessel, and/or the owner of another vessel on the node transferred ownership to the servicevessel pubkey, then `nodeman.cfg` would not reflect the new assignement, whereas `vesseldict` would, and the service logs would end up in the wrong vessel. (I don't think this has ever happened though. The clearinghouse does nothing like that.)

**Comment by JustinCappos** [comment]
We keep the node manager state in memory anyways in the nmmain's module
space.  So can't we just read it from there?

**Comment by aaaaalbert** [comment]
Yes, we can do that for nodemanager's use of servicelogger (assuming we refactor the `init` functions a bit).

A bigger problem exists in `repy.py`'s use of it (via `tracebackrepy`). `repy.py` doesn't load the NM config, yet some internal Repy errors could get logged in the nodemanager log, which is why `servicelogger` contains the logic to find the service vessel.

I think that the Repy sandbox should not take care of setting up log files itself, but rather use standard streams, and have the nodemanager redirect them to the appropriate places. I'll open another issue for this.

**Comment by JustinCappos** [comment]
What if this runs without a node manager?   For example, as a standalone
Repy program.

**Comment by aaaaalbert** [comment]
In the usual running-local case, `python repy.py restrictionsfile my_program.r2py`, the command-line arguments to use servicelogger aren't given. Thus, logging goes to `stdout`.
